_includes/app/txn-sample.py

```diff
@@ -26,33 +26,33 @@ def txnWrapper(conn, op):
                 onestmt(conn, "RELEASE SAVEPOINT cockroach_restart")
                 break
 
-            except psycopg2.DatabaseError as e:
-                if e.pgcode != '40001':
+            except psycopg2.OperationalError as e:
+                if e.pgcode != psycopg2.errorcodes.SERIALIZATION_FAILURE:
```

Comment:
If we're not going to suggest listening for our error `40001` error code, should I update step 4 [here](https://www.cockroachlabs.com/docs/transactions.html#transaction-retries) to be less exclusively focuses on that?